src/css_to_emotion.re

```diff
@@ -1156,6 +1155,26 @@ let render_style_rule = (ident, sr: Style_rule.t): expression => {
   );
 }
 
+
+let render_emotion_css = ((list, loc): Declaration_list.t, variables): expression => {
+  let declarationListValues = render_declaration_list((list, loc), variables);
+  let ident = Exp.ident(~loc, {txt: Emotion.lident("css"), loc});
+
+  Exp.apply(~loc, ident, [(Nolabel, declarationListValues)]);
+};
+
+let openEmotionWrapper = (~loc, expr) => {
```

Comment:
I removed openEmotionWrapper in previous PRs in favour of Emotion.lident, son instead of open inline we can use namespace: for example: `Emotion.css`